packages/auto-approve/src/auto-approve.ts

```diff
@@ -142,6 +142,14 @@ export function handler(app: Probot) {
       'pull_request.synchronize',
     ],
     async (context: Context<'pull_request'>) => {
+      // During codefreeze, simply set the RELEASE_FREEZE environment variable.
+      if (process.env.RELEASE_FREEZE) {
```

Comment:
How are we planning to set this env variable on deploy?

Reply:
Also do we only want to clock auto approve for release PRs (not dependency updates)?

Reply:
> Also do we only want to clock auto approve for release PRs (not dependency updates)?

This would be the ideal, but I think it's hard to distinguish today since the PRs we merge are based on title, committer, etc., not type of PR.

This will be easier in the future with some lovely refactoring @sofisl has planned.